docs/reference/errors-and-warnings/NU5130.md

```diff
@@ -0,0 +1,47 @@
+---
+title: NuGet Warning NU5130
+description: NU5130 Warning code
+author: zivkan
+ms.date: 09/16/2019
+ms.topic: reference
+ms.reviewer: karann
+f1_keywords:
+  - NU5130
+---
+
+# NuGet Warning NU5130
+
+<pre>Some target frameworks declared in the dependencies group of the nuspec and the lib/ref folder have compatible matches, but not exact matches in the other location. Unless intentional, consult the list of actions below:
+- Add a dependency group for .NETFramework4.7.2 to the nuspec</pre>
+
+### Issue
+
+The `lib/<tfm>/` or `ref/<tfm>/` directory in the package contains at least one file for the [Target Framework Moniker (TFM)](../target-frameworks.md) specified in the warning message. However, no dependency group exists for this TFM in the `nuspec` file. This may cause package consumers to believe the package is not compatible with the TFM. If the assemblies for the specified TFM have different dependencies to the compatible TFM defined in the dependencies group, the project using the package may experience runtime failures.
+
+### Solution
+
+* Run pack on the project
+
+If possible, use [NuGet's MSBuild pack task](../msbuild-targets.md), as it automatically matches assembly TFMs with dependency groups from the project's target frameworks. Note that your project must use `PackageReference` for its own NuGet dependencies. If your project uses packages.config, you need to use `nuget.exe pack` and a `nuspec` file.
```

Comment:
call it task or target in both if you accept @dominoFire's suggestion. 
Just be consistent :)